src/IO_Strings.f90

```diff
@@ -684,6 +683,14 @@ PURE FUNCTION nFields_char(aline)  RESULT(nfields)
             READ(aline(i+1:multidcol-1),*,IOSTAT=ioerr) nmult
             IF(ioerr /= 0) nmult=1
             n=n+(nmult-1)*2
+
+            !If we are multiplying a quoted string need to subtract 1.
```

Comment:
When you saying "multiplying a quoted string"...do you mean like 2*"some string"?

Reply:
yes. exactly.

Reply:
IF you look farther down near line 229 this test already existed (didn't need to add it). The problem was that the previously when that was made to work it broke the cases that were commented out.

I found an alternative way of implementing the routine such that both sets of ASSERTS could pass.

Make sense?

Reply:
I see...makes sense.